smarty/tests/test_atomtyper.py

```diff
@@ -15,7 +15,7 @@ def test_atomtyper(self):
         typetag = 'atomtype'
         atomtypes = AtomTyper.read_typelist(get_data_filename('atomtypes/basetypes.smarts'))
         replacements = AtomTyper.read_typelist(get_data_filename('atomtypes/replacements.smarts'))
-        molecules = read_molecules(get_data_filename('molecules/zinc-subset-tripos.mol2.gz'), verbose=False)
+        molecules = read_molecules('zinc-subset-tripos.mol2.gz', verbose=False)
```

Comment:
@davidlmobley 
If the file isn't local read_molecules looks in /openforcefield/data/molecules/